application/Core/UI/Properties/Resources.de-DE.resx

```diff
@@ -128,15 +128,23 @@
 Soll die Aufnahme gelöscht werden?</value>
   </data>
   <data name="Error_Configuration_Invalid" xml:space="preserve">
-    <value>Die Konfiguration konnte nicht korrekt geladen werden.
-Bitte überprüfen Sie die Konfiguration oder kontaktieren Sie einen Administrator.</value>
+    <value>Die Konfiguration scheint ungültig zu sein.
+Bitte überprüfen Sie die Konfiguration oder kontaktieren Sie einen Administrator.
+Vollständiger Fehler:
+</value>
   </data>
   <data name="Error_No_Configuration_Specified" xml:space="preserve">
     <value>Keine Konfigurationsdatei spezifiziert. Bitte spezifizieren Sie eine Konfigurationsdatei als Kommandozeilenparameter oder stellen Sie sicher, dass eine Datei mit dem Namen "config.morr" relativ zur Applikation existiert.</value>
   </data>
   <data name="Error_No_Recordings_Directory" xml:space="preserve">
     <value>Das Aufnahmeverzeichnis konnte nicht gefunden werden.
 Bitte kontaktieren Sie einen Administrator.</value>
+  </data>
+  <data name="Generic_Exception_During_Initialization" xml:space="preserve">
```

Comment:
When the configuration file is not found, a generic exception is thrown. While astute users can detect the actual reason by reading the first line of the exception message, it is not necessarily ideal. If a special error-message for the FileNotFoundException case is easily addable, i suggest doing so.

Reply:
Anything that has a distinct `Exception` subtype should be easily doable - I will add a message for `FileNotFoundException`. I don't think we will be able to catch all of those special cases though, even if you provide a path but that path is not a valid path, we throw an `ArgumentException` from the `FilePath` class, which I would recommend against trying to handle explicitly. If there are any other exceptions that you think we should and could handle explicitly though, please let me know.